tutorial/issue_tutorial.sh

```diff
@@ -0,0 +1,247 @@
+
+while true; do
+    echo "Do you want to send transaction to testnet or mainnet?"
+    echo "1 for testnet"
+    echo "2 for mainnet"
+
+    read env
+
+    case ${env} in
+        [1] )
+            echo "Ok, you choose testnet"
+            chain_id=${testnet_chain_id}
+            node=${testnet_node}
+            bnbcli=${testnet_bnbcli}
+            break;;
+        [2] )
+            echo "Ok, you choose mainnet"
+            chain_id=${mainnet_chain_id}
+            node=${mainnet_node}
+            bnbcli=${mainnet_bnbcli}
+            break;;
+        * ) echo 'Please input 1 or 2';;
+    esac
+done
+
+if [[ ! -f "${bnbcli}" ]]; then
+	echo "bnbcli does not exist!"
+	exit 1
+fi
+
+# Get account type
+while true; do
+    echo "Will you use Ledger or local key store to sign the transactions?"
+    echo "1 for legder"
+    echo "2 for local key store"
+
+    read account_type
+
+    case ${account_type} in
+        [1] )
+            echo "Ok, you choose legder"
+            break;;
+        [2] )
+            echo "Ok, you choose local key store"
+            break;;
+        * ) echo 'Please input 1 or 2';;
+    esac
+done
+
+# Get cli home
+while true; do
+    echo "Where is your bnbcli home?(default is ~/.bnbcli, just press enter if you want to use default)"
+    read bnbcli_home
+    if [[ "${bnbcli_home}" == "" ]]
+    then
+        bnbcli_home="${HOME}/.bnbcli"
+        echo "You choose default home!"
+    fi
+
+    if [[ ! -d "$bnbcli_home" ]]; then
+	    echo "bnbcli home(${bnbcli_home}) does not exist!"
+        continue
+    fi
+
+    if [[ ${account_type} == "1" ]]
+    then
+        accounts=$(echo "$(${bnbcli} keys list )" | grep ledger)
+        if [[ "${accounts}" == "" ]]
+        then
+            echo "there is no account exist"
+            continue
+        fi
+        echo "${accounts}"
+    else
+        accounts=$(echo "$(${bnbcli} keys list )" | grep local)
+        if [[ "${accounts}" == "" ]]
+        then
+            echo "there is no account exist"
+            continue
+        fi
+        echo "${accounts}"
+    fi
+    break
+done
+
+# Get account name
+while true; do
+    echo "Which account do you want to use?"
+    read account_name
+    if [[ "${account_name}" == "" ]]
+    then
+        echo "Please input account name"
+        continue
+    fi
+
+    result=$(echo "${accounts}" | awk '{print $1}' | grep "^${account_name}$")
+    if [[ "${result}" == "" ]]
+    then
+        echo "${account_name} does not exist"
+        continue
+    fi
+    break
+done
+
+# Get token name
+while true; do
+    echo "What is the token name?"
+    read token_name
+
+    result=$(echo ${token_name} | grep "^[0-9|a-z|A-Z| ]\{1,32\}$")
+    if [[ "$result" == "" ]]
+    then
+        echo "token name should only contains less than 33 alphanumeric characters and space, please try again"
+        continue
+    else
+        break
+    fi
+done
+
+# Get symbol
+while true; do
+    echo "What is the symbol?"
```

Comment:
maybe add the constrains: symbol should only contains 3~8 alphanumeric characters